conf/ocsci/ocp-4.4-config.yaml

```diff
@@ -0,0 +1,8 @@
+---
+# Use this config for installation of OCP 4.3 cluster
+RUN:
+  client_version: '4.4.0-0.nightly'
+DEPLOYMENT:
+  installer_version: "4.4.0-0.nightly"
+ENV_DATA:
+  vm_template: 'rhcos-4.3.0-x86_64-vmware'
```

Comment:
We should also use the 4.4 rhcos image.  We can take one from https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/4.4.0-0.nightly-2020-02-13-103342/

Reply:
changed to 'rhcos-4.4.0-x86_64-vmware'